### PR TITLE
OSDOCS-5563: add release note for LVMS configuration change

### PR DESCRIPTION
**Pull request by dhellmann**
Version(s): 4.13

Issue: [OSDOCS-5562](https://issues.redhat.com//browse/OSDOCS-5562)

Link to docs preview:
https://57829--docspreview.netlify.app/microshift/latest/microshift_release_notes/microshift-4-13-release-notes.html#microshift-4-13-storage

QE review: N/A

Additional information:

See #57417 for the original documentation update.

/assign @ShaunaDiaz

### CODE REVIEW

File: microshift_release_notes/microshift-4-13-release-notes.adoc [hunk 1]
```diff
@@ -59,8 +59,10 @@ This release adds improvements related to the following components and concepts.
 //[id="microshift-4-13-networking"]
 //=== Networking
```
Comment (ShaunaDiaz) [inline microshift_release_notes/microshift-4-13-release-notes.adoc:61]: This references OSDOCS-556**3**.

File: microshift_release_notes/microshift-4-13-release-notes.adoc [hunk 2]
```diff
@@ -59,8 +59,10 @@ This release adds improvements related to the following components and concepts.
...
 
-//[id="microshift-4-13-storage"]
-//=== Storage
+[id="microshift-4-13-storage"]
+=== Storage
+
+* {product-title} configures LVMS more dynamically based on the volume groups present on the system. For more details, refer to the xref:../microshift_storage/microshift-storage-plugin-overview.adoc#lvms-volume-group-name_{context}[LVMS system requirements] section of this documentation.
```
Comment (ShaunaDiaz) [inline microshift_release_notes/microshift-4-13-release-notes.adoc:65]:
```suggestion
* {product-title} configures LVMS more dynamically based on the Volume Groups present. For more details, read the xref:../microshift_storage/microshift-storage-plugin-overview.adoc#lvms-volume-group-name_{context}[LVMS system requirements] section of this documentation.
```

File: microshift_release_notes/microshift-4-13-release-notes.adoc (hunk 1 above)
Comment (dhellmann) [reply microshift_release_notes/microshift-4-13-release-notes.adoc:61]: Fixed the PR and commit message to point to the same ticket

**Comment by dhellmann** [comment]
@ShaunaDiaz I've updated to include your edits.

File: microshift_release_notes/microshift-4-13-release-notes.adoc [hunk 3]
```diff
@@ -59,8 +59,10 @@ This release adds improvements related to the following components and concepts.
 //[id="microshift-4-13-networking"]
 //=== Networking
 
-//[id="microshift-4-13-storage"]
-//=== Storage
+[id="microshift-4-13-storage"]
+=== Storage
+
+* {product-title} configures LVMS more dynamically based on the Volume Groups present. For more details, read the xref:../microshift_storage/microshift-storage-plugin-overview.adoc#lvms-volume-group-name_{context}[LVMS system requirements] section of this documentation.
```
Comment (ShaunaDiaz) [inline microshift_release_notes/microshift-4-13-release-notes.adoc:65]:
```suggestion
* {product-title} configures LVMS more dynamically based on the Volume Groups present. For more details, read the xref:../microshift_storage/microshift-storage-plugin-overview.adoc#microshift-lvms-system-requirements_{context}[LVMS system requirements] section of this documentation.
```

**Comment by dhellmann** [comment]
Updated and squashed

File: microshift_release_notes/microshift-4-13-release-notes.adoc [hunk 4]
```diff
@@ -59,8 +59,10 @@ This release adds improvements related to the following components and concepts.
 //[id="microshift-4-13-networking"]
 //=== Networking
 
-//[id="microshift-4-13-storage"]
-//=== Storage
+[id="microshift-4-13-storage"]
+=== Storage
+
+* {product-title} configures LVMS more dynamically based on the Volume Groups present. For more details, read the xref:../microshift_storage/microshift-storage-plugin-overview.adoc#microshift-lvms-system-requirements_{context}[LVMS system requirements] section of this documentation.
```
Comment (ShaunaDiaz) [inline microshift_release_notes/microshift-4-13-release-notes.adoc:65]:
```suggestion
* {product-title} configures LVMS more dynamically based on the Volume Groups present. For more details, read the xref:../microshift_storage/microshift-storage-plugin-overview.adoc#microshift-lvms-system-requirements[LVMS system requirements] section of this documentation.
```

Comment (ShaunaDiaz) [reply microshift_release_notes/microshift-4-13-release-notes.adoc:65]: [id="lvms-system-requirements_{context}"] That's the id of the module I can see in the 4.13 branch. The file name of the module was changed, but the id didn't get updated.

Comment (dhellmann) [reply microshift_release_notes/microshift-4-13-release-notes.adoc]: I think I worked out the right expanded form with the context value. Let's see how it works.

File: microshift_release_notes/microshift-4-13-release-notes.adoc [hunk 5]
```diff
@@ -59,8 +59,10 @@ This release adds improvements related to the following components and concepts.
 //[id="microshift-4-13-networking"]
 //=== Networking
 
-//[id="microshift-4-13-storage"]
-//=== Storage
+[id="microshift-4-13-storage"]
+=== Storage
+
+* {product-title} configures LVMS more dynamically based on the Volume Groups present. For more details, read the xref:../microshift_storage/microshift-storage-plugin-overview.adoc#lvms-volume-group-name_microshift-storage-plugin-overview[LVMS system requirements] section of this documentation.
```
Comment (mburke5678) [inline microshift_release_notes/microshift-4-13-release-notes.adoc:65]: Can you remove the _more_ here_? I tripped over _more dynamically based_, reading it as _more dynamically-based_ (if that makes sense).

**Comment by mburke5678** [comment]
@dhellmann One tiny nit. 
@ShaunaDiaz Where do you see that _Volume Group_ is capitalized? A quick search in Atom shows the _volume group_ predominantly used.

**Comment by ShaunaDiaz** [comment]
@mburke5678 I was looking at our parent assembly, the second-to-last sentence of the intro paragraphs:
"Each PVC references a storage class that represents an LVM Volume Group (VG) on the host node." 
A lot of this boiler-plate assembly text is _ye olde_ from OCP.

**Comment by dhellmann** [comment]
> @dhellmann One tiny nit.
> @ShaunaDiaz Where do you see that _Volume Group_ is capitalized? A quick search in Atom shows the _volume group_ predominantly used.

Done

**Pull request merged**